libs/ui/src/examples/Fetch/E1_FetchAxios.tsx

```diff
@@ -1,18 +1,33 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { useEffect, useState } from 'react'
 
-import { Box, Stack } from '@chakra-ui/react'
+import { Box, Stack, Text } from '@chakra-ui/react'
 import { API_URL, TOKEN } from '@wsvvrijheid/config'
+import axios from 'axios'
 
 const BLOG_URL = `${API_URL}/api/blogs` // https://api.samenvvv.nl/api/blogs
 const API_TOKEN = TOKEN
 
 export const FetcAxios = () => {
   const [blogs, setBlogs] = useState([])
```

Comment:
You can specify the type of state. Because we already know what it will be after fetching.

```suggestion
  const [blogs, setBlogs] = useState<Blog[]>([])
```